src/z3c/sqlalchemy/postgres.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Heh, when I said "at the top", I meant "right after the block comment".

(It's minor.)

Reply:
Here we go!
https://github.com/zopefoundation/z3c.sqlalchemy/pull/11